style.css

```diff
@@ -0,0 +1,111 @@
+h1 {
+    display: flex;
+    color: blue;
+    justify-content: center;
+}
+
+/* .mainWrapper {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+} */
+
+.quoteBanner {
+    display: inline-block;
+    text-align: center;
+}
+
+.allPicsDiv {                                  /* Div class for holding the first 3 pictures */
+    display: flex;
+    justify-content: space-around;
+    width: 75%;
+    margin: auto;
+}
+
+.allPicsDiv img{
+    display: flex;
+    align-self: center;
+    max-width: 100%;
```

Comment:
By allowing the pictures to expand to their full width you are loosing any space between them (as they expand to fill the flexbox). Try making them narrower to allow your flexbox to justify them with some space in between.